app/models/organization.rb

```diff
@@ -0,0 +1,68 @@
+require 'rest-client'
+require 'json'
+
+class Organization
+
+    attr_accessor :id, :name, :description, :imageUrl, :categories, :interests
+
+    def initialize(id, name, description, imageUrl, categories, interests)
+        @id = id
+        @name = name
+        @description = description
+        @imageUrl = imageUrl
+        @categories = categories
+        @interests = interests
+    end
+
+    def self.get_organizations(num, categories, interests)
+        orgs = self.get_orgs_from_api()
+	
+        top_orgs = []
+        orgs.each do |i|
+            if categories == nil || categories.size == 0 || categories.include?(i['type'])

```

Comment:
No organizations are being matched and aren't showing up in the `generate_orgs.html.erb` view because the categories from the API and the quiz differ in capitalization. This should work
```suggestion
            if categories == nil || categories.size == 0 || categories.include?(i['type'].titleize)
```